machine/v1/types_controlplanemachineset.go

```diff
@@ -32,6 +32,19 @@ type ControlPlaneMachineSet struct {
 
 // ControlPlaneMachineSet represents the configuration of the ControlPlaneMachineSet.
 type ControlPlaneMachineSetSpec struct {
+	// State defines whether the ControlPlaneMachineSet is Active or Inactive.
+	// When Inactive, the ControlPlaneMachineSet will not take any action on the
```

Comment:
What do you think about adding "controller" in these comments when you refer to CPMSO rather than just saying CPMS?
```suggestion
	// When Inactive, the ControlPlaneMachineSet controller will not take any action on the
```

Reply:
We discussed this offline and this matches with the MachineSet docs convention. So let's keep it this way.